src/vs/workbench/contrib/debug/browser/debugActionViewItems.ts

```diff
@@ -209,7 +209,13 @@ export class FocusSessionActionViewItem extends SelectActionViewItem {
 			}
 		}));
 
-		this._register(this.debugService.onDidNewSession(() => this.update()));
+		this._register(this.debugService.onDidNewSession(session => {
+			this._register(session.onDidChangeLabel(() => this.update()));
+			this.update();
+		}));
+		this.debugService.getModel().getSessions(true).forEach(session => {
```

Comment:
Please use `this.getSessions` instead.
We have that helper method since we have two action items and each displays different sessions. The one in the debug console displays inactive sessions, and the one in toolbar not (and it might dislpay subsessions depending on a setting)